core/indexer/processTransactions_test.go

```diff
@@ -481,3 +481,40 @@ func txPoolHasSearchOrder(txPool map[string]*Transaction, searchOrder uint32) bo
 
 	return false
 }
+
+func TestIsSCRForSenderWithGasUsed(t *testing.T) {
+	t.Parallel()
+
+	txHash := "txHash"
+	nonce := uint64(10)
+	sender := "sender"
+
+	tx := &Transaction{
+		Hash:   txHash,
+		Nonce:  nonce,
+		Sender: sender,
+	}
+	sc := ScResult{
+		Data:      []byte("@" + hex.EncodeToString([]byte("ok")) + "@something"),
```

Comment:
you can use directly `6f6b` instead of hex encoded `ok`. This way you simulate better the real scenario.

Reply:
small fix